### ISSUE TITLE
Greg Gould Testimony pdf 508 compliance check 

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/onrr.gov-site#750]
This request came in by email from Christina Mathers

Hi Maroya and Shannon, 

Please see attached 3 pdf of Greg Gould's testimony checked for accessibility. They should be ready to back on onrr.gov.  Please let me know if you have any questions. Thank you!
[FINA-SENR-ONRR-Testimony-on-Federal-Mineral-Revenues-(FINAL-07212014).pdf](https://github.com/ONRR/onrr.gov-site/files/6577122/FINA-SENR-ONRR-Testimony-on-Federal-Mineral-Revenues-.FINAL-07212014.pdf)
[Greg-GouldTestimonyHNR12-2015.pdf](https://github.com/ONRR/onrr.gov-site/files/6577124/Greg-GouldTestimonyHNR12-2015.pdf)

**Comment by mcharg** [comment]
Adding third file.[https://app.zenhub.com/files/210892300/9ccdd42f-e856-4450-ac06-7dfb24505c21/download](https://app.zenhub.com/files/210892300/9ccdd42f-e856-4450-ac06-7dfb24505c21/download)

**Comment by mcharg** [comment]
FINA-SENR-ONRR-Testimony-on-Federal-Mineral-Revenues-(FINAL-07212014).pdf has two errors, the other two look good.  Sent email to Christina to correct errors.

**Comment by mcharg** [comment]
The two accessible docs are updated in dev and sent Christina an email to approve.
